src/plugins/googledrive/server/services/operation-service.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+const path = require('path');
+const process = require('process');
+const { google } = require('googleapis');
+module.exports = ({ strapi }) => ({
+  get: async (ctx) => {
+    const SERVICE_ACCOUNT_PATH = path.join(
+      process.cwd(),
+      'config',
+      'env',
+      'google_drive_service_account.json'
+    );
+    const SCOPES = ['https://www.googleapis.com/auth/drive.metadata.readonly'];
+    const auth = new google.auth.GoogleAuth({
+      keyFile: SERVICE_ACCOUNT_PATH,
+      scopes: SCOPES,
+    });
+    const drive = google.drive({ version: 'v3', auth });
+
+    if (ctx.request.query.fileId === undefined) {
```

Comment:
If the field ID is not provided, return all files? This looks like a list endpoint instead of GET. Does our frontend need to list all files?

Reply:
The front-end Volunteer application form uses only the post and delete operations. Earlier I couldn't see the files that were uploaded through the post operation in the shared google drive. So I developed this list to view the files that were uploaded using the service account. I believe that may be useful.

Reply:
How is authentication handled? How do we make sure user A can't get application form of user B?

Reply:
Regarding the authentication the front-end calls the plugin. The plugin uses the service account information stored in configs.env file to upload the file. 

Currently the application does not have a feature to display the uploaded resume/portfolio.  The user can see the name of the uploaded file with a button to delete the file only  during the upload process/session. (Cannot see the contents of the file) If the user refreshes the page before submit there is no option to see and delete the file. After submitting, the files details are saved into the Applicants collection in Strapi. There is no screen to retrieve the application details that the user submitted atm.